torch/utils/_pytree.py

```diff
@@ -151,6 +148,9 @@ def tree_unflatten(values: List[Any], spec: TreeSpec) -> PyTree:
 
     return unflatten_fn(child_pytrees, spec.context)
 
+def tree_map(pytree: PyTree, fn: Any) -> PyTree:
```

Comment:
Could you swap these two arguments around? Python map takes in fn then args and since this is a map-like utility we should do the same